crates/forge/tests/e2e/running.rs

```diff
@@ -843,42 +843,60 @@ fn should_panic() {
     let snapbox = test_runner();
 
     let output = snapbox.current_dir(&temp).assert().code(1);
+
     assert_stdout_contains!(
         output,
         indoc! { r"
         [..]Compiling[..]
         [..]Finished[..]
 
 
-        Collected 6 test(s) from should_panic_test package
+        Collected 8 test(s) from should_panic_test package
         Running 0 test(s) from src/
-        Running 6 test(s) from tests/
-        [PASS] tests::should_panic_test::should_panic_no_data [..]
-
+        Running 8 test(s) from tests/
+        [FAIL] tests::should_panic_test::expected_panic_but_didnt
+        
+        Failure data:
+            Expected to panic but didn't
+        [PASS] tests::should_panic_test::should_panic_check_data (gas: ~1)
```

Comment:
```suggestion
        [PASS] tests::should_panic_test::should_panic_check_data [..]
```
Let's not assert gas in here

Reply:
Let's do it in the whole test*, sorry for not clarifying!